app/views/games/show.html.erb

```diff
@@ -1,9 +1,44 @@
-<h1>Game: <%= @game.id %></h1>
+<script>
+  $(function() {
+    $('.piece').draggable( {
+      containment: .board,
+      cursor: 'crosshair',
+      snap: false,
+      revert: invalidMove,
+    });
+    // function invalidMove(event, ui)
+
+    $('.board').droppable( {
+      tolerance: 'fit',
+      drop: dropConditions,
+      }
+      update: function(event, ui) {
+        $.ajax({
+          type: 'PUT',
+          url: ui.game.data('game-id'),
```

Comment:
x_coordinate/y_coordinate should come from the `droppable` square. (data attributes on the square)